thanks/index.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Sponsors
+---
+
+The wxWidgets project receives support through the donation of infrastructure from the companies listed below. We are very grateful for the use of these resources!
+
+Many companies and individuals make significant contributions time and human resources or support development through targeted funding; we intend to find a way to publicly acknowledge that in the near future.
+
+### Infrastructure Sponsors
+
+{::options parse_block_html="true" /}
+
+<div class="row">
+  <div class="col-sm-6 my-auto">
+[![GitHubLogo]][GitHub]
+  </div>
+  <div class="col-sm-6 my-auto">
+[![MacStadiumLogo]][MacStadium]
+  </div>
+</div>
+<div class="row">
+  <div class="col-sm-6 my-auto">
+[![TravisLogo]][Travis]
+  </div>
+  <div class="col-sm-6 my-auto">
+<h2 class="display-4">[AppVeyor][AppVeyor]{: class="text-reset"}</h2>
+  </div>
+</div>
+
+{::options parse_block_html="false" /}
+
+[GitHub]: https://github.com
+[GitHubLogo]: GitHub_Logo.png 
+{: width="200px"}
+
+[MacStadium]: https://www.macstadium.com
+[MacStadiumLogo]: https://uploads-ssl.webflow.com/5ac3c046c82724970fc60918/5c019d917bba312af7553b49_MacStadium-developerlogo.png 
```

Comment:
Perhaps we should copy their logo here too? This doesn't seem like a permanent/stable URL...

Reply:
good idea, thanks